### DISCUSSION TITLE
icon fix for VN (free video editing app)

### DISCUSSION THREAD

**Discussion by pindalol** [elrumo/macOS_Big_Sur_icons_replacements#1378] [Icon request]
Hi!
The VN app (downloadable for free on the app store) is a very simple and well done video editing for everyone is starting out on this field, and i love using it for every project not complicated enough to require premiere.
The only problem i have with this app is that the icon is slightly bigger than the others, it drives me mad to have it on the dock.

![Schermata 2022-05-04 alle 12 06 23](https://user-images.githubusercontent.com/104898986/166662131-3c4c30da-6200-4522-91de-bf796d3868c6.png)

Can please someone fix it? 
i'm totally fine with the aesthetics of the icon, it's just a bit too big 😣

Thank you in advance!

**Comment by OMGzilla-git** [comment]
Done. You can get it from my profile while it's awaiting approval.
https://macosicons.com/#/u/OMGzilla

**Reply by pindalol** [reply]
You're the GOAT! It's fantastic, thank you!
